lib/SILOptimizer/Differentiation/PullbackCloner.cpp

```diff
@@ -1804,6 +1804,11 @@ bool PullbackCloner::Implementation::run() {
   DominanceOrder domOrder(original.getEntryBlock(), domInfo);
   // Keep track of visited values.
   SmallPtrSet<SILValue, 8> visited;
+  // Also check for differentiable for all active values
```

Comment:
Not sure I understand this comment. The two lines of code below are just obtaining the protocol and the module and don't seem related to the comment.

Reply:
Right, this comment is more suitable next to the actual check below.